src/main/java/step3/domain/LottoUtil.java

```diff
@@ -0,0 +1,14 @@
+package step3.domain;
+
+public class LottoUtil {
+
+    private static final int LOTTO_PRICE = 1000;
+
+    private LottoUtil() {
+    }
```

Comment:
Reflection 을 통한 생성은 사실 일어날 일이 거의 없기 때문에 오버엔지니어링일 수 있습니다. 😅
저는 유틸리티 클래스의 생성자를 이렇게 구현하고 있다는 사실을 참고만 해주세요 🙇‍♂️🙏 
```suggestion
    private LottoUtil() {
        throw new AssertionError(String.format("%s class can not be instantiated", getClass().getName()));
    }
```